### ISSUE TITLE
nothing in main.rs

### ISSUE THREAD

**Issue by NPLeeWenKang** [grilledwindow/ratventure-rust#1]

**Comment by grilledwindow** [comment]
Hi @NPLeeWenKang, can you provide more details regarding this issue?

**Comment by NPLeeWenKang** [comment]
It has been a month since your init commit and main.rs only prints "hello world". I am starting to think that this repo is a scam.

**Comment by grilledwindow** [comment]
Yup, I have been a bit busy so I put off this project for a while. There's been some progress on the `feat/main-menu` branch, you can check that out.
